.github/workflows/build_and_deploy_search_bar.yml

```diff
@@ -0,0 +1,80 @@
+name: Build and deploy search bar
+
+on:
+  push:
+    tags:
+      - 'search-bar-v*'
+
+jobs:
+  call_build:
+    uses: ./.github/workflows/build.yml
+    with:
+      build_script: build-search-bar-only
+  
+  call_misc_tests:
+    uses: ./.github/workflows/miscellaneous_tests.yml
+
+  call_acceptance_search_bar:
+    uses: ./.github/workflows/acceptance_search_bar.yml
+    needs: call_build
+    secrets:
+      BROWSERSTACK_USERNAME: ${{ secrets.BROWSERSTACK_USERNAME }}
+      BROWSERSTACK_ACCESS_KEY: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
+
+  call_extract_versions:
+    uses: ./.github/workflows/extract_versions.yml
+    with:
+      ignore_prefix: search-bar-
+
+  call_should_deploy_major_version:
+    uses: ./.github/workflows/should_deploy_major_version.yml
+    with:
+      ignore_prefix: search-bar-
+
+  call_deploy_tag_version:
```

Comment:
nit: can we rename to call_deploy_full_version the two deploy and hold workflow? mainly since we ignore prefix 'search-bar-' in tag version for this case

Reply:
Sure, I can rename. Are you suggesting to rename it in this search bar workflow and in the regular build_and_deploy_hold workflow?

Reply:
yea